tests/integration/playbooks/rollback.yaml

```diff
@@ -0,0 +1,75 @@
+---
+- hosts: guest
+  become: false
+  vars:
+    total_counter: "0"
+    failed_counter: "0"
+
+  tasks:
+    - name: rpm-ostree rollback
+      command: rpm-ostree rollback
```

Comment:
This is OK for now but I think in the future we should be pointing at https://github.com/containers/bootc/issues/383 - worth a cross-reference.

Reply:
I saw this issue last week. So I think I should add `rpm-ostree rollback` first. And it will be changed to `bootc rollback` in PR which added this feature.